src/sfizz/FilePool.cpp

```diff
@@ -412,7 +411,7 @@ void sfz::FilePool::emptyFileLoadingQueues() noexcept
 {
     emptyQueue = true;
     while (emptyQueue)
-        std::this_thread::sleep_for(1ms);
+        std::this_thread::sleep_for(std::chrono::microseconds(100));
```

Comment:
The time interval was changed here. Is it intentional?

Reply:
If it was because of me, it is certainly not intentional. good catch btw